### PR TITLE
upgrade 3.1/4.0 minor cs fixes

### PR DESCRIPTION
**Pull request by HeahDude**
| Q | A |
| --- | --- |
| Branch? | master |
| License | MIT |

### CODE REVIEW

File: UPGRADE-3.1.md [hunk 1]
```diff
@@ -23,10 +24,11 @@ FrameworkBundle
 ---------------
 
  * As it was never an officially supported feature, the support for absolute
-   template paths has been deprecated and will be removed in Symfony 4.0.
+   template paths in the template name parse has been deprecated and will be
+   removed in Symfony 4.0.
```
Comment (fabpot) [inline UPGRADE-3.1.md]: should be reverted

File: UPGRADE-4.0.md [hunk 2]
```diff
@@ -15,10 +15,16 @@ Form
...
-   `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed
+   `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed.
 
- * The following form types registered as services has been removed; use
-   their fully-qualified class name instead:
+FrameworkBundle
+---------------
+
+ * Support for absolute template paths has been removed from the template name
+   parser.
```
Comment (fabpot) [inline UPGRADE-4.0.md]: It's really not just about the parser. Full support is removed.

Comment (HeahDude) [inline UPGRADE-4.0.md]: I am not aware of this change. What should be the sentence though:
`Support for absolute template paths has been removed` ?

Comment (fabpot) [inline UPGRADE-4.0.md]: right

**Comment by HeahDude** [comment]
Ok fixed!

**Comment by fabpot** [comment]
Thank you @HeahDude.

**Pull request merged**